Traceback (most recent call last):
  File "/Users/kootsoop/opt/anaconda3/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/kootsoop/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/kootsoop/opt/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/kootsoop/opt/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/kootsoop/opt/anaconda3/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/kootsoop/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/kootsoop/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/kootsoop/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import schemdraw
import schemdraw.elements as elm
with schemdraw.Drawing(file='voltage-between-non-reference-nodes-currents.svg') as d:
    d.push()
    d += elm.GroundSignal()
    d.pop()
    d += (R1 := elm.Resistor().label('$R_1$\n 3$\Omega$').up())
    d += elm.Dot(color='red').label('A', loc='top', color='black')
    d += (R2 := elm.Resistor().label('$R_2$\n 6$\Omega$').right())
    d += elm.Dot(color='red').label('B', loc='top', color='black')
    d.push()
    d += (R3 := elm.Resistor().label('$R_3$\n 12$\Omega$').down())
    d.pop()
    d += (VS := elm.SourceV().label('$V_S$\n8$~V$').right().reverse())
    d += elm.Dot(color='red').label('C', loc='top', color='black')
    d += elm.CurrentLabelInline(direction='in', ofst=-0.6).at(VS.start).label('$I_O$')
    d.push()
    d += (R4 := elm.Resistor().label('$R_4$\n 4$\Omega$').down())
    d.pop()
    d += (R5 := elm.Resistor().label('$R_5$\n 8$\Omega$').right())
    d += elm.Dot(color='red').label('D', loc='top', color='black')
    d += (R6 := elm.Resistor().label('$R_6$\n 5$\Omega$').down())
    d += elm.Line().left().length(9)
    d.move_from(R1.end, 0.5, -1)
    d += elm.Arrow().down().length(1).color('blue')
    d.move_from(R2.start, 1, -0.5)
    d += elm.Arrow().right().length(1).color('blue')
    d.move_from(R3.start, 0.5, -1)
    d += elm.Arrow().down().length(1).color('blue')
    d.move_from(R4.start, 0.5, -1)
    d += elm.Arrow().down().length(1).color('blue')
    d.move_from(R6.start, 0.5, -1)
    d += elm.Arrow().down().length(1).color('blue')
    d.move_from(R5.start, 1, -0.5)
    d += elm.Arrow().right().length(1).color('blue')
    d.move_from(R2.end, -0.5, -0.6)
    d += elm.Line().tox(R5.start + 0.5).color('orange')
    d += elm.Line().up().color('orange').length(2.5)
    d += elm.Line().left().color('orange').length(4).label('Super Node')
    d += elm.Line().down().color('orange').length(2.5)
```

`````{admonition} Example

Find $I_O$ using nodal analysis
```{figure} voltage-between-non-reference-nodes.svg
---
height: 300px
name: voltage-between-non-reference-nodes
---
------------------

[0;36m  File [0;32m"/var/folders/65/qy_bbqpj2x3470cgr0fmhyc40000gn/T/ipykernel_41596/2571915734.py"[0;36m, line [0;32m41[0m
[0;31m    ```[0m
[0m    ^[0m
[0;31mSyntaxError[0m[0;31m:[0m invalid syntax

SyntaxError: invalid syntax (2571915734.py, line 41)

